cabal-install/src/Distribution/Client/ProjectPlanning/Types.hs

```diff
@@ -643,7 +675,7 @@ data ElaboratedComponent
     compComponentName :: Maybe ComponentName,
     -- | The *external* library dependencies of this component.  We
     -- pass this to the configure script.
-    compLibDependencies :: [ConfiguredId],
+    compLibDependencies :: [(ConfiguredId, Bool)],
```

Comment:
Same here, what does the 'Bool' mean.

Additionally, are these really *external* library dependencies?